.eslintrc

```diff
@@ -10,5 +10,6 @@
     "no-mixed-operators": [2, { "allowSamePrecedence": true }],
     "jsx-a11y/no-static-element-interactions": 1, // TODO: enable
     "react/no-did-mount-set-state": 0, // necessary for server-rendering
+    "linebreak-style": 0, //  this is to prevent windows/unix conflicts on linebreaks-style (LF -> CRLF)
```

Comment:
why is this here? on windows you should be using gitattributes to ensure everything is always LFs.